### ISSUE TITLE
Entries disappear and problem with saving

### ISSUE THREAD

**Issue by gwbonline** [bolt/bolt#2608]
I have installed Bolt 2.0.5 and I have now some strange things happening. When I want to use the 'Save and go back to overview' function, Bolt stays in the edit window. After using the same function again, Bolt goes to the overview of entries. I use Firefox, it also happens in IE.

The second strange things is the following. With the help of Bob, our website has a page with all broadcasts displayed (title, day and time) with the code below. When I open an entry and save it again, the entry goes away on the page I described here. What goes wrong? I don't understand this...

```
<p>{% set days = ['Maandag', 'Dinsdag', 'Woensdag', 'Donderdag', 'Vrijdag', 'Zaterdag', 'Zondag'] %} 
                                                 
 {% for dag in days %} 
                                   
  {% setcontent uitzending = "progs" where { 'dag': '%'~dag~'%' }  %}

                                   <b>{{ dag }}</b><br />
  {% for record in uitzending|order('tijdstip') %}
    {{ record.tijdstip }} <a href="{{ record.link }}" rel="bookmark">{{ record.title }}</a><br />
  {% endfor %}
{% endfor %}
```

**Comment by bobdenotter** [comment]
Ugh, I have a faint idea what's happening here. Could you post the relevant part (with the `days`) from your `contenttypes.yml` here?

**Comment by gwbonline** [comment]
dag:
           type: select
           label: Dag
           values: [ Maandag, Dinsdag, Woensdag, Donderdag, Vrijdag, Zaterdag, Zondag ]
        tijdstip:
            type: select
            label: Tijdstip
            values: [ 09:00, 10:00, 11:00, 12:00, 13:00, 14:00, 15:00, 16:00, 17:00, 18:00, 19:00, 20:00, 21:00, 22:00, 23:00 ]

**Comment by bobdenotter** [comment]
It's as i expected! Two things you can do: Wait for the 2.0.6 update (soon!) in which it'll be fixed. Alternatively, make the keys in your array specific for now. Like this: 

```
    dag:
       type: select
       label: Dag
       values:
           "Maandag": "Maandag"
           "Dinsdag": "Dinsdag"
           "Woensdag": "Woensdag"
           "Donderdag": "Donderdag"
           "Vrijdag": "Vrijdag"
           "Zaterdag": "Zaterdag"
           "Zondag": "Zondag"
    tijdstip:
        type: select
        label: Tijdstip
        values: 
            "09:00": "09:00"
            "10:00": "10:00"
            "11:00": "11:00"
            "12:00": "12:00"
            "13:00": "13:00"
            "14:00": "14:00"
            "15:00": "15:00"
            "16:00": "16:00"
            "17:00": "17:00"
            "18:00": "18:00"
            "19:00": "19:00"
            "20:00": "20:00"
            "21:00": "21:00"
            "22:00": "22:00"
            "23:00": "23:00"
```

For reference, these ate the issues where it was fixed: #2609 and #2597

**Comment by gwbonline** [comment]
Thanks! I'll wait for the update then. Is the problem with 'saving and 
go back to overview' also fixed?
